src/deviceApiCalls/transports/extension.transport.js

```diff
@@ -0,0 +1,7 @@
+import {DeviceApiTransport} from '../../../packages/device-api'
+
+export class ExtensionTransport extends DeviceApiTransport {
+    async send (deviceApiCall) {
+        throw new Error('not implemented yet for ' + deviceApiCall.method)
```

Comment:
This is because I suppose we haven't ported the extension calls to use the transport. Was there a specific challenge/reason for it or was it just descoped?